WordPress/gutenberg
# Rollback navigation areas

## Description

Related to https://github.com/WordPress/gutenberg/issues/36524. This PR rolls back navigation areas.

## Test steps

Confirm that all editors load properly, and the navigation block is being saved and rendered as expected.

## Concerns

This block was shipped in Gutenberg 11.9, and now it's being reverted. It is hard to provide a fallback, too – it would essentially mean retaining the block. If Gutenberg is an "early preview" of the next WordPress, it should be okay to revert this PR. Still, I would like to get more opinions on that: @getdave @draganescu @noisysocks @talldan @mtias  @youknowriad @jsnajdr @youknowriad @spacedmonkey @tellthemachines @azaozz @hellofromtonya.
